frontend/src/api/PartnerAPI.js

```diff
@@ -16,4 +16,29 @@ function createPartner(credentials) {
   return api.post('/partners', credentials)
 }
 
-export { updatePartner, getPartners, getPartner, createPartner }
+async function inactivatePartner(id) {
+  try {
+    return await api.put(`/partners/${id}/setstatus`)
```

Comment:
Mi a különbség az `await api.put(`/partners/${id}/setstatus`)` és az `await api.put(`/partners/${id}/setstatus`)` között? Ha ez a setstatus végpont (amit semmiképpen nem így hívnék amúgy) esetleg egy toggle jellegű működést valósít meg, akkor azt nem ajánlom, mert nem idempotens. Ez az jelenti, hogy ha netán kétszer küldöm be a kérést, akkor is ugyanazt a végeredményt kellene kapnom. Kapcsolgatós működés esetén ez nem így van...
Miért küldeném be kétszer? Mondjuk két admin van és pont egyszerre akarják aktiválni. Mindkettő azt látja, hogy inaktív, mindkettő rákattint, hogy aktiválás és ha idempotens, akkor a felhasználó aktív lesz, ha toggle-os a működés, akkor marad inaktív.

Reply:
Backenden Toggle-s volt az endpoint, javítottam.